documentation/examples/11-easing/00-easing/Controls.svelte

```diff
@@ -46,6 +47,7 @@
 		{:else}
 			<ul>
 				{#each types as [name, type]}
+				<!-- svelte-ignore a11y-click-events-have-key-events a11y-no-noninteractive-element-interactions -->
```

Comment:
same comment as above - create a button inside the `<li>` instead of ignoring the warnings